regparser/tree/depth/derive.py

```diff
@@ -145,6 +147,25 @@ def derive_depths(original_markers, additional_constraints=[]):
     return solutions
 
 
+def derive_depths_relaxed(original_markers, additional_constraints=[]):
+    """Derive paragraph depth with reduced constraints,
+    including removal of emphasis and disabling of the
+    depth_type_inverses optional rule.
+    """
```

Comment:
This is misleading -- this function doesn't do everything you are describing here, it's really only removing emphasis. I'd recommend moving the deemphasis logic out one level.

Related: #178 adds a `deemphasize` function, so you'd be able to just
```python
markers = [deemphasize(m) for m in markers]
```